sktime/regression/deep_learning/cntc.py

```diff
@@ -250,6 +252,7 @@ def _predict(self, X, **kwargs):
         preds = self.model_.predict([X2, X, X], self.batch_size, **kwargs)
         preds = np.squeeze(preds, axis=-1)
         return preds
+    
```

Comment:
this new newline will cause formatting tests to fail, please fix

see here how to automate formatting locally https://www.sktime.net/en/stable/developer_guide/coding_standards.html